ige/CMakeLists.txt

```
@@ -98,6 +98,7 @@ set(sources ${sources}
     "include/ige/plugin/InputPlugin.hpp"
     "include/ige/plugin/physics/PhysicsWorld.hpp"
     "include/ige/plugin/physics/RigidBody.hpp"
+    "include/ige/plugin/physics/GhostObject.hpp"
```

Keep the file list sorted alphabetically
```suggestion
    "include/ige/plugin/physics/GhostObject.hpp"
    "include/ige/plugin/physics/PhysicsWorld.hpp"
    "include/ige/plugin/physics/RigidBody.hpp"
```